assets/model_monitoring/components/src/data_drift_compute_metrics/compute_data_drift.py

```diff
@@ -62,10 +64,14 @@ def compute_data_drift_measures_tests(
             categorical_columns_names,
             categorical_threshold,
         )
-
+        print(categorical_df)
     # TODO: fix this if, else
     if len(numerical_columns_names) != 0 and len(categorical_columns_names) != 0:
-        output_df = numerical_df.union(categorical_df)
+        if categorical_df is not None:
+            output_df = numerical_df.union(categorical_df)
+        else:
+            print("categorical_df is none")
+            output_df = numerical_df
     elif len(numerical_columns_names) != 0:
         output_df = numerical_df
     else:
```

Comment:
what if both of them are None

Reply:
then we will not have columns is my understanding because we wont be able to get datatype. This check is covered